orquesta/conducting.py

```diff
@@ -564,12 +574,13 @@ def get_next_tasks(self):
                 elif 'items_count' in next_task and next_task['items_count'] == 0:
                     next_tasks.append(next_task)
             except Exception as e:
+                fail_on_task_rendering = True
                 self.log_error(e, task_id=staged_task['id'], route=staged_task['route'])
                 self.request_workflow_status(statuses.FAILED)
```

Comment:
Could line 579 be moved to under line 583, it will be called only once.

Reply:
I think you're correct. Let me move it and see.